docquality.rst

```diff
@@ -32,8 +32,8 @@ and validates that your new markup is correct.
 
 You can view the documentation built from :ref:`in-development <indevbranch>`
 and :ref:`maintenance <maintbranch>` branches at https://docs.python.org/dev/.
-The in-development and most recent 3.x (as well as 2.x) maintenance
-branches are rebuilt once per day.
+The in-development and most recent 3.x maintenance branches are rebuilt once per
+day.
```

Comment:
Would it hurt readability if the `3.x` was also removed?

```suggestion
The in-development and recent maintenance branches are rebuilt once per day.
```

Reply:
This is better, thanks.